Source/Core/Core/HW/DVDThread.cpp

```diff
@@ -47,9 +47,9 @@ static u32 s_output_address;
 static u32 s_length;
 static bool s_decrypt;
 
-// Used to notify emulated software after executing command.
-// Pointers don't work with savestates, so CoreTiming events are used instead
-static int s_callback_event_type;
+// This determines which function will be used as a callback.
+// We can't have a function pointer here, because they can't be in savestates.
+static bool s_reply_to_ios;
```

Comment:
It may not be particularly wise to store a bool into the savestate, since its size can vary across platforms (I realize this is done with s_decrypt, but just bringing it up as just-in-case).